traitlets/traitlets.py

```diff
@@ -1644,6 +1644,21 @@ def trait_events(cls, name=None):
                         events[k] = v
         return events
 
+    def clear_traits(self, *names, **metadata):
```

Comment:
I think it's better to start as many (probably all) trait-related methods with `trait_`. We've been much too lax adding to the namespace of user objects with new methods, and I think a pseudo-namespace of a `trait_` prefix lowers the cost to new inherited methods because they are grouped together.